pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [project]
-name = "devpy"
-version = "0.2rc1.dev0"
+name = "spin"
+version = "0.2b1"
```

Comment:
Just for my own education, what's the beta1 about?

Reply:
Whoops, not sure why I left that there...